spec/IS20.md

```diff
@@ -51,24 +70,35 @@ type Metadata = {
    name : text; // token name
    symbol : text; // token symbol
    decimals : nat8; // token decimal
-   totalSupply : nat; // token total supply
+   totalSupply : Tokens128; // token total supply
    owner : principal; // token owner
    logo : text; // base64 encoded logo or logo url
-   fee : nat; // fee for update calls
+   fee : Tokens128; // fee for update calls
 }
 ```
 
```

Comment:
mint(to: AccountIdentifier, value){
account_balances[accID] += value 
}

transfer_from_any(x) {
// while loop 
}

fn transfer(){
//check that the accID formed from caller() has balance
}

fn transfer_toaccountid(accid val) {
mintees[accID] = value 
// store principal in the tx
}

fn